pages/pipelines/managing_log_output.md

```diff
@@ -146,7 +146,7 @@ For example, if you have environment variable `MY_SECRET="topsecret"`and you run
 By default, build logs are stored in encrypted form in Buildkite's managed Amazon S3 buckets, but you can instead store the archived build logs in your private AWS S3 bucket.
 
 >📘
-> This feature is only available to customers on the Buildkite <a href="https://buildkite.com/pricing">Enterprise</a> plan.
+> This feature is only available to customers on the Buildkite <a href="https://buildkite.com/pricing">Enterprise</a> plan and is applied at organisation level. Enterprise customers with multiple organisations need to send a list to which organisation to enable this feature.
 
```

Comment:
Adding content from the note below:

````suggestion

The folder structure and file format are as follows and are not customizable: 

```text
{BUILDKITE_PIPELINE_ID}/{BUILDKITE_BUILD_ID}/{BUILDKITE_JOB_ID}.log
```

````

Reply:
Applied locally and pushed